rotkehlchen/db/upgrades/v36_v37.py

```diff
@@ -42,25 +42,46 @@ def _update_history_events_schema(write_cursor: 'DBCursor', conn: 'DBConnection'
 
         for entry in read_cursor:
             if entry[11] is None:
-                new_entries.append(entry[:11] + ('none',) + entry[12:])
+                new_entries.append(entry[:11] + ('none',))
             else:
-                new_entries.append(entry)
+                new_entries.append(entry[:12])
```

Comment:
this [:12] looks prone to error. Aren't you using a length constant somewhere else already?

Reply:
Using a length constant is less safe imo since the constant will be changing over time when we add / remove fields from history base entry. 12 is the number of needed fields at the point when the upgrade is happening.

Reply:
Pleases write is a contant with a comment

Reply:
Ok